s3fs/core.py

```diff
@@ -296,6 +296,7 @@ def __init__(
         asynchronous=False,
         loop=None,
         max_concurrency=1,
+        fixed_upload_size: int | None = None,
```

Comment:
Could i be a bool parameter instead, so that we have only one "size" to worry about?

Reply:
done